resources/snap_packs/snap_packs.yml

```diff
@@ -0,0 +1,353 @@
+post:
+  operationId: snap_pack_create
+
+  summary: Create
+
+  description: >-
+    Creates a new snap_pack given information
+
+  tags:
+    - Snap Packs
+
+  parameters:
+    - $ref: "../../shared/parameters/idempotency.yml#/idem-header"
+    - $ref: "../../shared/parameters/idempotency.yml#/idem-query"
+    - $ref: "parameters/lob_version.yml#/lob-version"
```

Comment:
Referring to the local reference vs the shared lob-version reference - the verbiage needs to be slightly different